download_py_package.py

```diff
@@ -35,6 +38,30 @@
 
 def download_py_package():
     """Download package which needs to be analyzed by future steps."""
+    page = request.urlopen(os.path.join(Configuration.PACKAGE_INDEX, Configuration.PACKAGE_NAME))
+    version_exists = 0
```

Comment:
`True`/`False`? They map to ints in Python, but it might be easier to read the code.